libs/langchain/langchain/document_loaders/directory.py

```diff
@@ -55,6 +60,8 @@ def __init__(
             show_progress: Whether to show a progress bar. Defaults to False.
             use_multithreading: Whether to use multithreading. Defaults to False.
             max_concurrency: The maximum number of threads to use. Defaults to 4.
+            sample_size: The maximum number of files you would like to load from the directory.
+            randomize_sample: Suffle the files to get a random sample.
```

Comment:
Could you add documentation for sample_seed

Reply:
Sorry for missing that sure thing.